coderd/notifications/testdata/rendered-templates/TemplateTemplateDeleted-body.md.golden

```diff
@@ -1,3 +1,5 @@
-Hi Bobby
+Hi Bobby,
 
-The template **bobby-template** was deleted by **rob**.
\ No newline at end of file
+The template **bobby-template** was deleted by **rob**.
+
+The template's display name was **Bobby's Template**.
```

Comment:
The display name is just a human-readable representation of the template name, so in this case, you don't need to indicate that it is not the real template name.

Just go with:

> The template **Bobby's Template** was deleted by **rob**.

Reply:
I wondered about this, but didn't want to assume I could overwrite what was specified.
Will update.